docs/nci_gadi.md

```diff
@@ -0,0 +1,73 @@
+# nf-core/configs: NCI Gadi HPC Configuration
+
+nf-core pipelines have been successfully configured for use on the [Gadi HPC](https://opus.nci.org.au/display/Help/Gadi+User+Guide) at the National Computational Infrastructure (NCI), Canberra, Australia.
+
+To run an nf-core pipeline at NCI Gadi, run the pipeline with `-profile singularity,nci_gadi`. This will download and launch the [`nci_gadi.config`](../conf/nci_gadi.config) which has been pre-configured with a setup suitable for the NCI Gadi HPC cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+## Access to NCI Gadi
+
+Please be aware that you will need to have a user account, be a member of an Gadi project, and have a service unit allocation to your project in order to use this infrastructure. See the [NCI user guide](https://opus.nci.org.au/display/Help/Getting+Started+at+NCI) for details on getting access to Gadi.
+
+## Launch an nf-core pipeline on Gadi
+
+### Prerequisites
+
+Before running the pipeline you will need to load Nextflow and Singularity, both of which are globally installed modules on Gadi. You can do this by running the commands below:
+
+```bash
+module purge
+module load nextflow singularity
+```
+
+### Execution command
+
+```
```

Comment:
```suggestion
```bash
```
Gives nice code highlighting and the copy code button.

Reply:
Thanks @mashehu have incorporated this and all suggestions below in my fixes :)